ui/src/forms/editors/DictionaryTableEditor.vue

```diff
@@ -39,18 +36,18 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, PropType } from '@vue/composition-api'
+import { defineComponent, PropType } from 'vue'
 import { PropertyObjectState } from '@hydrofoil/shaperone-core/models/forms'
 import { prov, sh } from '@tpluscode/rdf-ns-builders'
 import { Term } from 'rdf-js'
 import { PropertyShape } from '@rdfine/shacl'
 import { PropertyRenderer } from '@hydrofoil/shaperone-core/renderer'
 import RadioButton from '@/components/RadioButton.vue'
-import RenderWcTemplate from '../RenderWcTemplate.vue'
+import DictionaryTableEditorPair from './DictionaryTableEditorPair.vue'
```

Comment:
Why did you add an additional component if it's used only once here?

Reply:
In order to make the `templateResult` a computed property instead of a function call in the template itself. The function call was causing the template to re-render in a loop.